modepy/modes.py

```diff
@@ -126,9 +126,11 @@ def jacobi(alpha, beta, n, x):
     PL = np.zeros((Nx, n+1), np.float64)  # noqa: N806
 
     # Initial values P_0(x) and P_1(x)
-    gamma0 = (2**(alpha+beta+1)
-            / (alpha+beta+1.)
-            * gamma(alpha+1) * gamma(beta+1) / gamma(alpha+beta+1))
+    if abs(alpha + beta + 1) < 1.0e-14:
+        gamma0 = np.pi
```

Comment:
Could you add a comment on what's wrong below without this conditional?

(Maybe also mention that this case covers Chebyshev of the first kind?)

Reply:
Fixed in d21df36. Also extended the formula so that it actually works for all `alpha + beta == 1` cases, not just Chebyshev.